frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -133,18 +130,10 @@ const isDefaultSnapshotClass = (volumeSnapshotClass: VolumeSnapshotClassKind) =>
   ] === 'true';
 
 const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
-  const {
-    resourceName,
-    plural,
-    namespace,
-    kindObj,
-    handlePromise,
-    inProgress,
-    errorMessage,
-  } = props;
+  const { pvcName, namespace, handlePromise, inProgress, errorMessage } = props;
```

Comment:
ping @spadgett 
I've simplified this file by removing async loading of Kind data, because it seems to me that it's not necessary. As far as I can tell, it works as I would expect. However I'm not sure if I'm missing something about the reason why we would load it asynchronously rather than using the existing constants.